### PR TITLE
Consolidating Cron Notes

### PR DESCRIPTION
**Pull request by JGrubb**
Also adding a note about jitter.

### CODE REVIEW

File: src/configuration/app-containers.md [hunk 1]
```diff
@@ -521,7 +519,17 @@ crons:
         cmd: 'cd web ; drush queue-run aggregator_feeds'
 ```
 
-Also note that all server times are in UTC, and hence so are the cron schedules.
+> ** Cron notes **
```
Comment (Crell) [inline src/configuration/app-containers.md]: I don't think we even need to put this in a notes box.  It's just more info about cron, let it be prose.

Comment (JGrubb) [inline src/configuration/app-containers.md]: Ha, ok.  My eye grepped for "note" and landed on that.

Comment (Crell) [inline src/configuration/app-containers.md]: I meant any of it. :smile: I'm not generally a fan of callout boxes.

File: src/configuration/app-containers.md [hunk 2]
```diff
@@ -521,7 +519,17 @@ crons:
...
+> 
+> Cron runs are executed using the dash shell, not the bash shell used by normal SSH logins. 
+> In most cases that makes no difference but may impact some more involved cron scripts.
+>
+> A variable delay is added to each cron job in each project in order to prevent host overloading
+> at midnight every night when every project runs their nightlies.  Your crons will *not* run exactly
+> at the time that you specify, but will be delayed by 0-300 seconds.
+
+All server times are in UTC, and hence so are the cron schedules.
```
Comment (ralt) [inline src/configuration/app-containers.md]: You can specify the timezone for the cron, no?

Comment (Crell) [inline src/configuration/app-containers.md]: Can you?  If you know how that's a good thing to include. :smile:

Comment (ralt) [inline src/configuration/app-containers.md]: It will be whatever you put in the `.platform.app.yaml`'s `timezone` key.

Comment (Crell) [inline src/configuration/app-containers.md]: The timezone key is not currently documented anywhere in the file. *sigh*  Let's get back to that later.

**Pull request merged**